cloudplaya/client.py

```
@@ -329,14 +339,17 @@ def _load_session(self):
         session = json.loads(f.read())
         f.close()
 
-        if 'cp_token' not in session:
+        try:
+            self.customer_id = session['customer_id']
+            self.csrf_rnd = session['csrf_rnd']
+            self.csrf_token = session['csrf_token']
+            self.csrf_ts = session['csrf_ts']
+            self.device_id = session['device_id']
+            self.device_type = session['device_type']
+            self.cookies = session['cookies']
+        except KeyError:
```

We should be logging what exactly happened and make it clear that there was a failure to authenticate. Right now, we're just silently dropping any problems on the floor.